.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
+
+on:
+    push:
+      branches:
+        - develop
+    workflow_dispatch:
+permissions:
+  contents: write
+
+jobs:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install and Deploy 🔧
+        run: |
+          npm install
+          npm run build
+
+      - name: Deploy 🚀
```

Comment:
why not use `npm run deploy`?

Reply:
docusarus deploy requires some env variables. why not just go with build and deploy?